assets/queries/terraform/aws/rds_database_cluster_not_encrypted/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	rds := input.document[i].resource.aws_db_cluster_snapshot[name]
+	db := rds.db_cluster_identifier
+	dbName := split(db, ".")[1]
+
+	not rds_cluster_encrypted(dbName)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_db_cluster_snapshot[%s].db_cluster_identifier", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "aws_db_cluster_snapshot.db_cluster_identifier' is encrypted",
+		"keyActualValue": "aws_db_cluster_snapshot.db_cluster_identifier' is not encrypted",
+		"searchLine": common_lib.build_search_line(["resource", "aws_db_cluster_snapshot", name, "db_cluster_identifier"], []),
```

Comment:
Based on [@rogeriopeixotocx requested changes](https://github.com/Checkmarx/kics/pull/4163/files#r705145894), this should change to:
```suggestion
		"searchLine": common_lib.build_search_line(["resource", "aws_db_cluster_snapshot", name], []),
```